packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -629,36 +630,41 @@ export default {
 
         /**
          * Check form is valid - no inline messages
-         * If form is valid try to call `submitCheckout`
-         * Catch and handle any errors
+         * 1. If form is valid try to call `submitCheckout`.
+         * 2. If the form is invalid process error tracking and logging via `onInvalidCheckoutForm()`.
          */
         async onFormSubmit () {
             this.trackFormInteraction({ action: 'submit' });
 
             if (!this.isFormValid()) {
```

Comment:
I'm wondering if this might be tidier inverted?

```javascript
if (this.isFormValid()) {
    await this.submitCheckout();
} else {
    this.onInvalidCheckoutForm();
}
```

Reply:
I think that makes sense as it was confusing me at the start too. Will fix monday.

Reply:
Fixed.